server/controllers/article.controller.js

```diff
@@ -204,5 +204,26 @@ class ArticleController {
     await sendMail(mailOption);
     return Response(res, 200, 'Article shared successfully');
   }
+
+  /**
+   * Delete article
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} response
+   */
+  static async deleteArticle(req, res) {
+    const {
+      params: { slug }, user: { id }
+    } = req;
+
+    const article = await Article.destroy({
+      where: {
+        slug,
+        userId: id,
+      }
+    });
+    if (article) return Response(res, 200, `Successfully deleted ${article} article`);
```

Comment:
Good job @Chiazokam, I think this line will be okay without returning the count of articles deleted that is `Successfully deleted article`, or I will need clarification if this endpoint can handle multiple delete. what do you think about this?

Reply:
I thought it wise to give the user an idea of what happened behind, hence the count